apy/anki.py

```diff
@@ -320,8 +320,11 @@ def list_cards(self, query, verbose=False):
         """List cards that match a query"""
         for cid in self.find_cards(query):
             c = self.col.getCard(cid)
-            question = html_to_screen(c.q()).replace('\n', ' ')
-            answer = html_to_screen(c.a()).replace('\n', ' ')
+            # Use the note raw values instead of the templated values c.q() and c.a()
+            raw_question, raw_answer = c.note().values()
+            question = html_to_screen(raw_question).replace('\n', ' ')
+            answer = html_to_screen(raw_answer).replace('\n', ' ')
```

Comment:
These 3 lines are the fix for #34. I separated this from the contribution code in 2 different commits, in case you disagree with one or the other and would like to revert a commit.